protocol-units/da-sequencer/node/src/celestia/client.rs

```diff
@@ -1,29 +1,25 @@
 use super::submit::BlobSubmitter;
-use super::{
-	BlockSource, CelestiaBlobData, CelestiaClientOps, CelestiaHeight, ExternalDaNotification,
-};
-use crate::block::SequencerBlockDigest;
+use super::{BlockSource, CelestiaBlob, CelestiaClientOps, CelestiaHeight, ExternalDaNotification};
 use crate::error::DaSequencerError;
-
 use celestia_rpc::Client as RpcClient;
 use celestia_types::nmt::Namespace;
+use movement_types::block;
+use std::sync::Arc;
```

Comment:
Nit: I usually put `std` imports into their own group so they stand out better. But if this is how rustfmt automatically rearranges it, OK.

Reply:
At some point, we need to create a document with coding conventions.

Reply:
A style guide such as 
https://github.com/paritytech/substrate/blob/master/docs/STYLE_GUIDE.md

Reply:
Yes, and further restrictions. Blank lines between `use` declarations prevent `rustfmt` from sorting. We should keep them as tidy as possible (sorting + merging).